src/main/kotlin/com/openlattice/hazelcast/serializers/RemoveMembersFromOrganizationAssemblyProcessorStreamSerializer.kt

```diff
@@ -29,15 +29,14 @@ import com.openlattice.assembler.processors.RemoveMembersFromOrganizationAssembl
 import com.openlattice.hazelcast.StreamSerializerTypeIds
 import com.openlattice.organizations.SecurablePrincipalList
 import org.springframework.stereotype.Component
-import javax.inject.Inject
 
 @Component
 class RemoveMembersFromOrganizationAssemblyProcessorStreamSerializer
     : SelfRegisteringStreamSerializer<RemoveMembersFromOrganizationAssemblyProcessor>,
       AssemblerConnectionManagerDependent<Void?> {
     private lateinit var acm: AssemblerConnectionManager
-    @Inject
-    private lateinit var splss: SecurablePrincipalListStreamSerializer
+
+    private val splss = SecurablePrincipalListStreamSerializer()
```

Comment:
probably fine for now but I could see us wanting a singleton for every `StreamSerializer`. 
Actually shouldn't this `StreamSerializer` have a static method to write/read and therefore not need an instance? 🤔

Reply:
it should -- actually yeah I think this was meant to be a temporary/quick fix to get prod back up, but we should add static serialize/deserialize methods in SecurablePrincipalListStreamSerializer and use those instead

Reply:
👍

Reply:
Actually, that is harder than expected, since it's implementing `ListStreamSerializer`

Reply:
It's fine for now but I think `CollectionStreamSerializer` should implement `TestableSelfRegistering` and then the implementation should be passed all the way up